### PR TITLE
RNs 39649 + 39632

### PR DESCRIPTION
**Pull request by EdithLansens**
@stephenlanszweertsky, could you please review this pull request to confirm the accurate inclusion of release note 39649 in the user guide? Thanks in advance!

### CODE REVIEW

**Comment by EdithLansens** [comment]
Hi @WoutMH, I've also added some changes to this pull request based on RN 39632. Could you double-check this pull request as well? Thanks in advance.

File: user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Low_Code_Apps/Low-Code_Apps_configuration/LowCodeApps_event_config.md [hunk 1]
```diff
@@ -106,6 +106,9 @@ To configure an event to close all panels of the app:
 
 - Select *Close all panels*.
 
+> [!NOTE]
+> Prior to DataMiner 10.3.0 [CU16]/ 10.4.0 [CU4]/10.4.7, navigating to a different low-code app page would close any open panels. From DataMiner 10.3.0 [CU16]/ 10.4.0 [CU4]/10.4.7 onwards, panels remain open when navigating to a different page. To ensure older apps function correctly after upgrading to DataMiner 10.3.0 [CU16]/ 10.4.0 [CU4]/10.4.7 or higher, an *On close* page event with a *Close all panels* action is automatically configured for each page of apps created before the upgrade<!--RN 39632-->. New apps created after the upgrade will not have these events configured.
+
```
Comment (WoutMH) [inline user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Low_Code_Apps/Low-Code_Apps_configuration/LowCodeApps_event_config.md:111]: Also the case for new pages added after the upgrade

Comment (EdithLansens) [reply user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Low_Code_Apps/Low-Code_Apps_configuration/LowCodeApps_event_config.md:111]: Added in [commit 68b1dd0](https://github.com/SkylineCommunications/dataminer-docs/pull/3266/commits/68b1dd0810b4f4acb4a55b162ae9470ad04fb212).

**Review by WoutMH — APPROVED** [review]

**Review by stephenlanszweertsky — APPROVED** [review]

**Comment by MariekeGO** [comment]
@EdithLansens, I've reviewed the PR and made some changes. Could you double-check before merging?

**Pull request merged**